src/main/server/adapter/ceasn/ceasn.js

```diff
@@ -389,10 +380,25 @@ async function cassFrameworkAsCeasn() {
     var ctx = JSON.stringify((await httpGet("https://credreg.net/ctdlasn/schema/context/json"))["@context"],true);
     const terms = JSON.parse(JSON.stringify((await httpGet("https://schema.cassproject.org/0.4/jsonld1.1/cass2ceasnTerms")),true));
     f.competency = [];
+    let mappedCompetencies = [];
     for (let i = 0; i < allCompetencies.length; i+=100) {
-        await Promise.all(allCompetencies.slice(i, i+100).map((id) => competencyPromise(id, competencies, allCompetencies, f, ctx, terms)));
+        let batch = await Promise.all(allCompetencies.slice(i, i+100).map((id) => competencyPromise(id, competencies, allCompetencies, f, ctx, terms)));
+        mappedCompetencies.push(... batch); 
+    }
+    console.log(mappedCompetencies);
```

Comment:
Either remove or change to use the new auditLogger format if its important info to log, `global.auditLogger.report(global.auditLogger.LogCategory.ADAPTER, global.auditLogger.Severity.INFO,  "ceasnExportUriTransform", mappedCompetencies)`